validations/schema_validate_test.go

```diff
@@ -267,3 +268,36 @@ lastName: World`)
 		Ω(getPropByName(node, "x")).Should(BeNil())
 	})
 })
+
+var _ = DescribeTable("sort validation issues", func(lines []int) {
+	var issues YamlValidationIssues
+	if lines == nil {
+		issues = nil
+	} else {
+		issues = make(YamlValidationIssues, len(lines))
+		for i, line := range lines {
+			issues[i] = YamlValidationIssue{Line: line, Msg: fmt.Sprintf("line %d issue", line)}
+		}
+
+		issues.Sort()
+
+		// Check it's sorted
+		isSorted := sort.SliceIsSorted(issues, func(i, j int) bool {
+			return issues[i].Line < issues[j].Line
+		})
+		Ω(isSorted).Should(Equal(true), fmt.Sprintf("slice is not sorted: %v", issues))
+
+		// Check the values are correct
+		for _, issue := range issues {
```

Comment:
sorry, I didn't get a point of this check. Do you mean that sort should not change the message of issue?
don't we want to check they are actually sorted?

Reply:
We checked the sort order in the previous Ω, this loop checks that the messages are still reported on the correct line (i.e. we didn't just change the line numbers to be sorted, but actually sorted the issues)